src/lib/airtable/patches/patch-01.patch

```diff
@@ -6,7 +6,7 @@ index 7b68055..25078e7 100644
    getRecordById,
    deleteRecord,
  } from './airtable';
-+import { addToOfflineCustomer } from '../utils/customerUtils';
++import { addToOfflineCustomer } from '../utils/offlineUtils';
```

Comment:
This change was made to patch-01 because a previous PR changed the name of `customerUtils` to `offlineUtils`